superset-frontend/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -35,6 +35,7 @@ import {
   setInLocalStorage,
 } from 'src/utils/localStorageHelpers';
 import { URL_PARAMS } from 'src/constants';
+import cls from 'classnames';
```

Comment:
The codebase has been using `cx` as the import for this module.

Reply:
Done